lib/trino-orc/src/main/java/io/trino/orc/metadata/OrcMetadataWriter.java

```diff
@@ -50,12 +50,12 @@
         implements MetadataWriter
 {
     // see https://github.com/trinodb/orc-protobuf/blob/master/src/main/protobuf/orc_proto.proto
-    private static final int TRINO_WRITER_ID = 4;
+    public static final int TRINO_WRITER_ID = 4;
```

Comment:
revert? i think we should check for `!= hive` rather than `== trino`?

Reply:
Not convinced. We know what Trino does. And we do not know what other writers do. It is safer to expect other writers to explicitly tell us what orc acid version is. And fail if they do not tell us. If we tested for != hive we would continue with reading files which have unknown format.